features/gezag-minderjarige/2a.1-zijn-juridische-ouders-nu-met-elkaar-gehuwd-of-partners.feature

```diff
@@ -51,12 +51,17 @@ Functionaliteit: 2a.1 - Zijn juridische ouders nu met elkaar gehuwd of partners
       zijn.
 
   Gebruikte velden:
-    - BSN ouder 1 -> 02.01.20 
-    - BSN ouder 2 -> 03.01.20
-    - BSN in huwelijk/geregistreerd partnerschap -> 05.01.20
-    - Datum huwelijkssluiting/aangaan geregistreerd partnerschap -> 05.06.10
-    - Datum ontbinding huwelijk/geregistreerd partnerschap -> 05.07.10
-    - Reden huwelijksontbinding -> 05.07.40
+    - Geboortedatum persoon -> 01.03.10
+    - Burgerservicenummer ouder 1 -> 02.01.20
+    - Geslachtsnaam ouder 1 -> 02.02.40
+    - Burgerservicenummer ouder 2 -> 03.01.20
+    - Geslachtsnaam ouder 2 -> 03.02.40
+    - Burgerservicenummer huwelijk/geregistreerd partnerschap -> 05.01.20
+    - Start datum huwelijk/geregistreerd partnerschap -> 05.06.10
```

Comment:
historiche huwelijk of partnerschappen missen in de lijst

Reply:
Historie in de context van huwelijk of geregistreerd partnerschap (HOP) is een beetje dubbel. Iedere HOP heeft zijn eigen historie (gebaseerd op volg nummer). Ieder persoon kan een of meerdere ex-partners hebben (gebaseerd op stapel nummer). In principe zijn wij alleen geïnteresseerd in "historie" op basis van stap nummer, niet in de "historie" op basis van volg nummer. Hoe willen we dit detail documenteren?

Alle andere `Geschiedenis*` domein objecten die we gebruiken om historische gegevens te representeren zijn trouwens gebaseerd op volg nummer.

Reply:
ja duidelijk, neem ik mee met https://github.com/BRP-API/brp-api-gezag/issues/260